pyinfraformat/core/io.py

```diff
@@ -1,15 +1,15 @@
 # pylint: disable=try-except-raise
 """Input and output methods."""
+import json
```

Comment:
Should we require ujson?

For large files, it is much much faster

Reply:
1000 holes takes
wfs request: 643 ms ± 143 ms per loop
json load: 8.97 ms ± 606 µs per loop

Time benefit would be negligible with ujson. Intersting package though

Reply:
true, let's postpone it. It is more important with files and 1GB+